d2game/d2player/escape_menu.go

```diff
@@ -360,6 +362,7 @@ func (m *EscapeMenu) OnLoad() {
 }
 
 func (m *EscapeMenu) OnEscKey() {
+	// note: original D2 returns straight to the game from however deep in the menu we are
```

Comment:
Good catch. I ported this behavior over to `OnEscKey` without noticing this. Want to assign me an issue to fix?